### DISCUSSION TITLE
rpc error: code = DeadlineExceeded desc = context deadline exceeded

### DISCUSSION THREAD

**Discussion by jarac794** [websu-io/websu#17] [General] [answered]
I am trying to analyze a website and it throws me this error, would you know a possible solution, I am using docker: 8000

rpc error: code = DeadlineExceeded desc = context deadline exceeded

**Comment by samos123** [comment, marked as answer]
Just in case someone arrives here, we discussed this already here: https://github.com/websu-io/websu/issues/20
